### DISCUSSION TITLE
Legacy Update not working correctly on Windows XP Service pack 3

### DISCUSSION THREAD

**Discussion by Silcat7794** [LegacyUpdate/LegacyUpdate#168] [Questions & Help] [answered]
Hello! I was using legacy update to update my Win XP laptop to SP3, and it worked fine. Once it updated me to IE8, nothing would work. I tried compatibility mode, and it worked. I came back later, used compatibility mode, but now it doesn't work. What do it do? Thanks!

P.S. I've included a picture of what happens when I try to load legacy update webpage.
![20230814_173906(1)](https://github.com/kirb/LegacyUpdate/assets/142269074/84642086-b7c5-406a-8999-0356b461c743)

**Comment by Silcat7794** [comment]
Hello? Anyone there?

**Reply by Silcat7794** [reply]
Update: I've done my own searching, and I find this in developer tools. I'm not a pro at this stuff, so I don't know what this means, but I'm assuming it means that Legacy Update expects IE 5, but IE 8 is present. If someone could confirm this, please message back. Thank you.
![20230821_143832](https://github.com/kirb/LegacyUpdate/assets/142269074/50f0dddd-a114-4404-96d2-fee5938406dc)

**Reply by kirb** [reply]
Don't worry about the classes, those are used for styling (CSS). What matters is the browser mode in the top bar, which is correct being IE8.

Can you see any warnings/errors in the Script tab? Refreshing the page while that tab is open may show a bit more useful info.

**Reply by Silcat7794** [reply]
I looked through the script, I did not see any errors. Is there a specific part of the script that there should be an error, that I didn't see, or something else?

**Reply by Silcat7794** [reply]
Oh wait a minute, I think I found something. 
![20230822_091554](https://github.com/kirb/LegacyUpdate/assets/142269074/d2b1a35f-5b13-40e4-a5b7-b23d4fca71c2)

**Reply by kirb** [reply]
Sorry, I should have specifically said on the right hand side, where it displays a log of any messages/errors/etc seen while the code is executing. I'm guessing you probably didn't see anything interesting there then?

**Reply by Silcat7794** [reply]
No, the other side is empty. I think for some reason the website is having trouble detecting what OS I'm running, because the first time this happened I tried compatibilty mode and it worked, it showed the OS I was running at the top right, then started checking for updates. now, it doesn't show anything.

**Reply by Silcat7794** [reply]
Sorry to keep bothering you, but are you there?

**Reply by kirb** [reply]
I’m really sorry but I don’t have a good answer for why that’s happening to you at the moment. I’ve only seen it mentioned by one other person on the Discord (who might have been you?), and I haven’t been able to reproduce the bug myself. Best as I can suggest is to delete temporary files (Safety &rarr; Delete Browsing History &rarr; tick Temporary Internet Files) and refresh, see if that makes any difference.

**Reply by Silcat7794** [reply]
Nope, I don't have discord. I'll try that and see if it works, thanks!

**Reply by nbuly11** [reply, marked as answer]
I know I am a year late BUT I had the same issue and deleting my temporary internet files worked! So if anyone else has this same issue, listen to kirb. Thanks a lot!

**Reply by Silcat7794** [reply]
I think I might've tried that, but glad it worked for you! I just ended up reinstalling XP, and it worked again. Thanks for replying though! 😁
